src/main/java/uk/co/ramp/event/AlertEventProcessor.java

```diff
@@ -120,6 +117,6 @@ int timeInStatus(AlertStatus newStatus) {
             .mean(progressionData.mean())
             .max(progressionData.max())
             .build();
-    return distributionSampler.getDistributionValue(distribution);
+    return distributionSampler.getDistributionValue(distribution) * properties.timeStepsPerDay();
```

Comment:
I thought the idea was that we kept the disease settings in days, then scaled everything else. I think it might be cleaner to change the MeanMax interface to be doubles rather than ints so we can add finer granularity there.

Reply:
Sure that sounds right. Sounds good to me.